libs/fn/src/lib/segmented-button/selectable-button.directive.ts

```diff
@@ -0,0 +1,93 @@
+import {
```

Comment:
it should be named using `segmented-button` in name to be consisted with the component name, otherwise there may be missunderstanding

Reply:
`segmented-button-item` or something